sdk/tests/deqp/functional/gles3/es3fNegativeTextureApiTests.js

```diff
@@ -488,7 +490,7 @@ goog.scope(function() {
 
 
             bufferedLogToConsole('gl.INVALID_VALUE is generated if imageSize is not consistent with the format, dimensions, and contents of the specified compressed image data.');
-            gl.compressedTexImage2D(gl.TEXTURE_2D, 0, gl.COMPRESSED_RGBA8_ETC2_EAC, 0, 0, 0, null);
+            gl.compressedTexImage2D(gl.TEXTURE_2D, 0, gl.COMPRESSED_RGBA8_ETC2_EAC, 0, 0, 0, new Uint8Array(1));
```

Comment:
The original deqp test uses -1 here, which is impossible for WebGL.

Uint8Array(1) is a reasonable replacement of -1.